incubator/hnc/api/v1alpha2/hnc_config.go

```diff
@@ -95,7 +95,6 @@ type CodeAndAffectedNamespaces struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:path=hncconfigurations,scope=Cluster
-// +kubebuilder:skipversion
```

Comment:
Does this need to be removed from v1a1 as well?

Reply:
This is not added in any v1a1. The other two places that it still remains are v1a2/ subnamespaceanchor.go and hierarchy_type.go. For these two other CRDs, we will still skip the v1a2 before there's any conversion webhook is there.